shoop/themes/classic_gray/static_src/js/custom.js

```diff
@@ -29,6 +29,29 @@ function getProductListViewMode() {
     return "grid";
 }
 
+function updateProductPrice(productId) {
+    const $quantity = $("#product-quantity");
+    if (!$quantity.is(":valid")) {
+        return;
+    }
+    var data = {
```

Comment:
If we can use const binds, this could also be one (you're never reassigning `data`, just editing it)